docs/installation.rst

```diff
@@ -102,16 +119,6 @@ Running the ingestion tasks
   The "-B" option tells the celery worker to startup a beat service, so that periodic tasks can be executed.
   You only need one worker with the beat service enabled. Multiple beat services will result in periodic tasks being executed multiple times
 
```

Comment:
We should mention the new ingest_push command here. Basically, instead of running a full ingestion task, you can process just the jobs associated with a single push in a synchronous manner. This is ideal for testing.

The execution syntax is just:

./manage.py ingest_push <project> <revision hash>